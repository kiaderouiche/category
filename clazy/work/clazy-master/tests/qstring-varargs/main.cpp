#include <QtCore/QString>

void simple_printf(const char* ...)
{
}


void test()
{
    QString s;
    printf("%s", s); // Warn
    simple_printf("%s", s); // Warn
    const char *foo = "f";
    printf("%s", foo);

    QByteArray b;
    printf("%s", b); // Warn
}
