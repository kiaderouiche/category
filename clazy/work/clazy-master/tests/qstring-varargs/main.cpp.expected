qstring-varargs/main.cpp:11:18: warning: Passing QString to variadic function [-Wclazy-qstring-varargs]
qstring-varargs/main.cpp:12:25: warning: Passing QString to variadic function [-Wclazy-qstring-varargs]
qstring-varargs/main.cpp:17:18: warning: Passing QByteArray to variadic function [-Wclazy-qstring-varargs]
